docs/development/extensions-core/kinesis-ingestion.md

```diff
@@ -557,7 +557,7 @@ fail-overs.
 A supervisor is stopped via the `POST /druid/indexer/v1/supervisor/<supervisorId>/terminate` endpoint. This places a
 tombstone marker in the database (to prevent the supervisor from being reloaded on a restart) and then gracefully
 shuts down the currently running supervisor. When a supervisor is shut down in this way, it will instruct its
-managed tasks to stop reading and begin publishing their segments immediately. The call to the shutdown endpoint will
+managed tasks to stop reading and begin publishing its segments immediately. The call to the shutdown endpoint will
```

Comment:
"publishing _their_ segments" was the correct version, I believe. The tasks are supposed to publish segments, not the supervisor.

Reply:
Thanks! I split it into to phrases to make that clear :)